addons/full-import/export.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+source /usr/local/pf/addons/full-import/helpers.functions
+source /usr/local/pf/addons/full-import/database.functions
+
+output="$1"
+
+if [ -z "$output" ]; then
+  echo "Missing output parameter"
+  echo "Usage export.sh /path/to/export.tgz [--force]"
+  exit 1
+fi
+
+db_dump=`find /root/backup/ -name 'packetfence-db-dump-*' -mtime -1 | tail -1`
+
+if [ -z "$db_dump" ]; then
+  echo "Unable to find a database dump that was done in the last 24 hours. Add --force to ignore this."
+  exit 1
+fi
+
+files_dump=`find /root/backup/ -name 'packetfence-files-dump-*' -mtime -1 | tail -1`
+
+if [ -z "$files_dump" ]; then
+  echo "Unable to find a files dump that was done in the last 24 hours. Add --force to ignore this."
+  exit 1
+fi
+
+build_dir=`mktemp -d`
```

Comment:
I suggest to use `export_dir` in place of `build_dir`

Reply:
I create a new package called `packetfence-export` for v10.3 (Deb and RPM).

This package is a standalone package not dependent to `packetfence` package. It should only be installed on PacketFence installations before 11.0.

This package is built on each commit on devel (can be improve later) and uploaded automatically to Debian and RPM repositories on inverse.ca. A copy to a friendly name like for `packetfence-release` package is done to:
- https://www.packetfence.org/downloads/PacketFence/RHEL8/
- https://www.packetfence.org/downloads/PacketFence/debian/

For 11.0 and next releases, `full-import` directory is part of `packetfence` package.